modules/machineset-yaml-rhv.adoc

```diff
@@ -113,16 +115,20 @@ If you are using a version earlier than {rh-virtualization} 4.4.8, see link:http
 
 <16> Optional: Specify the size of the bootable disk in GiB.
 
-<17> Optional: List of the network interfaces of the VM. If you include this parameter, {product-title} discards all network interfaces from the template and creates new ones.
+<17> Optional: Specify the UUID of the storage domain for the compute node's disks. If none is provided, the compute node is created on the same storage domain as the control nodes. (default)
 
-<18> Optional: Specify the vNIC profile ID.
+<18> Optional: Specify `true` to set the disk to preallocated or `false` if it should remain as thin (default).
```

Comment:
`false` if it should be thin-provisioned (default)

Reply:
Fixed.